arches/app/utils/file_validator.py

```diff
@@ -14,15 +15,17 @@ def __init__(self):
 
     def test_unknown_filetypes(self, file, extension=None):
         errors = []
-        if extension in settings.FILE_TYPES:
-            if extension == "xlsx":
+        match extension:
+            case "DS_Store":
+                self.logger.log(logging.WARN, "DS_Store file encountered, proceeding with caution.")
```

Comment:
These get skipped later in `validate_file_type()`, but adding some future safety here in case that ever changes, or there are other callers to this method.